inputSchema/wellCompletions.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
I guess the location for this file is ok for now, but we probably need to change it if we split the component. We'll probably have to see if we will have it with the component, with the backend or in a separate package.

Reply:
yes, I agree